Forms/Schema/FormSchema.php

```diff
@@ -65,7 +69,8 @@ public function getState(Form $form) {
 
 		if($form->Message()) {
 			$state['messages'][] = [
-				'value' => $form->Message(),
+				// TODO Make form / field messages not always stored as html
+				'value' => ['html' => $form->Message()],
```

Comment:
yeah, I think keeping this consistent is important, I had left this as `messages` for if more messages were to be added - but that can be for another story.